packages/nodes-base/nodes/Zammad/descriptions/UserDescription.ts

```diff
@@ -0,0 +1,658 @@
+		],
+	},
+	{
+		displayName: 'Update Fields',
+		name: 'updateFields',
+		type: 'collection',
+		displayOptions: {
+			show: {
+				operation: [
+					'update',
+				],
+				resource: [
+					'user',
+				],
+			},
+		},
+		default: {},
+		placeholder: 'Add Field',
+		options: [
+			{
+				displayName: 'Active',
+				name: 'active',
+				type: 'boolean',
+				default: false,
+			},
+			{
+				displayName: 'Address',
+				name: 'addressUi',
+				type: 'fixedCollection',
+				default: {},
+				options: [
+					{
+						displayName: 'Address Details',
+						name: 'addressDetails',
+						values: [
+							{
+								displayName: 'City',
+								name: 'city',
+								type: 'string',
+								default: '',
+								placeholder: 'Berlin',
+							},
+							{
+								displayName: 'Country',
+								name: 'country',
+								type: 'string',
+								default: '',
+								placeholder: 'Germany',
+							},
+							{
+								displayName: 'Street & Number',
+								name: 'address',
+								type: 'string',
+								default: '',
+								placeholder: 'Borsigstr. 27',
+							},
+							{
+								displayName: 'Zip Code',
+								name: 'zip',
+								type: 'string',
+								default: '',
+								placeholder: '10115',
+							},
+						],
+					},
+				],
+			},
+			{
+				displayName: 'Custom Fields',
+				name: 'customFieldsUi',
+				type: 'fixedCollection',
+				default: '',
+				placeholder: 'Add Custom Field',
+				typeOptions: {
+					multipleValues: true,
+				},
+				options: [
+					{
+						name: 'customFieldPairs',
+						displayName: 'Custom Field',
+						values: [
+							{
+								displayName: 'Field',
+								name: 'name',
+								type: 'options',
+								typeOptions: {
+									loadOptionsMethod: 'loadUserCustomFields',
+								},
+								default: '',
+								description: 'Name of the custom field to set',
+							},
+							{
+								displayName: 'Value',
+								name: 'value',
+								type: 'string',
+								default: '',
+								description: 'Value to set on the custom field',
+							},
+						],
+					},
+				],
+			},
+			{
+				displayName: 'Department',
+				name: 'department',
+				type: 'string',
+				default: '',
+				placeholder: 'Finance',
+			},
+			{
+				displayName: 'Email Address',
+				name: 'email',
+				type: 'string',
+				default: '',
+				placeholder: 'hello@n8n.io',
+			},
+			{
+				displayName: 'Fax',
+				name: 'fax',
+				type: 'string',
+				default: '',
+				placeholder: '+49 30 901820',
+			},
+			{
+				displayName: 'First Name',
+				name: 'firstname',
+				type: 'string',
+				default: '',
+				placeholder: 'John',
+			},
+			{
+				displayName: 'Last Name',
+				name: 'lastname',
+				type: 'string',
+				default: '',
+				placeholder: 'Smith',
+			},
+			{
+				displayName: 'Notes',
+				name: 'note',
+				type: 'string',
+				default: '',
+				typeOptions: {
+					alwaysOpenEditWindow: true,
+				},
+			},
+			{
+				displayName: 'Organization Name/ID',
+				name: 'organization',
```

Comment:
Because the posted basic auth creds expired. Confirmed working with new basic auth creds. Either switch to token auth or use the new basic auth creds posted in the ticket or create new ones.